installer.sh

```diff
@@ -406,19 +406,38 @@ case ${ANSWER:0:1} in
 esac
 
 # aha for html generation - future extension of emba
-#echo -e "\\n""$ORANGE""$BOLD""Downloading aha""$NC"
-#if ! [[ -f "external/aha" ]] ; then
-#  apt-get install make
-#  wget https://github.com/theZiz/aha/archive/master.zip -O external/aha-master.zip
-#  unzip ./external/aha-master.zip -d ./external
-#  rm external/aha-master.zip
-#  cd ./external/aha-master || exit 1
-#  echo -e "$ORANGE""$BOLD""Compile aha""$NC"
-#  make
-#  cd ../.. || exit 1
-#  mv "external/aha-master/aha" "external/aha"
-#  rm -R external/aha-master
-#else
-#  echo -e "$ORANGE""aha is already downloaded and compiled""$NC"
-#fi
+
+INSTALL_APP_LIST=()
+echo -e "\\nTo use the aggregator and check if exploits are available, we need a searchable exploit database. CVE-searchsploit will be installed via pip3."
```

Comment:
something weird is happening here. This code was already in the installer. Why is it marked as new code?